build-system/eslint-rules/await-expect.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = function(context) {
```

Comment:
Can you add a good and bad comment to the file? Eg https://github.com/ampproject/amphtml/blob/5f75efe35b734a4fcf0884d373315de60cd42df9/build-system/eslint-rules/no-deep-destructuring.js#L21-L24